Support/lib/rspec/mate/switch_command.rb

```diff
@@ -140,8 +140,9 @@ def klass(relative_path, content=nil)
       def write_and_open(path)
         FileUtils.mkdir_p(File.dirname(path))
         described = described_class_for(path, base_dir)
+        is_rails = File.directory? path.sub(%r{/spec/.*$}, '/app')
```

Comment:
I think this check may be too unspecific … are there other frameworks also using a top-level `app` directory? But right now I can’t think of a good alternative either (apart from parsing `Gemfile.lock` to see if it include `rails`) … any ideas?